jira_integration_example/utilities/secrets.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Google, LLC.
```

Comment:
Same idea for pubsub.py applies to secrets.py: I looked into symlinks and found that it wouldn't work since Docker doesn't support symlinking to files outside the build context, so I am going to continue with duplicating the pubsub.py and secrets.py for each integration example for the time being.